api/layout/paths.go

```diff
@@ -37,25 +37,24 @@ func EntriesPathForLogIndex(seq, logSize uint64) string {
 	return EntriesPath(seq/256, logSize)
 }
 
-// EntriesPath returns the local path for the nth entry bundle. p denotes the partial
-// tile size, or 0 if the tile is complete.
-func EntriesPath(n, logSize uint64) string {
+// NWithSuffix returns a tiles-spec "N" path, with a partial suffix if applicable.
+func NWithSuffix(l, n, logSize uint64) string {
```

Comment:
Is there any logic to what function returns with a leading slash, and which one don't?
For instance, "EntriesPath" doesn't start with a leading one, but "fmtN" does. It's slightly orthogonal to this change, but it would be good to make things consistent / document them / use path.Join if it makes sense